plugins/foreman_discovery/index.md

```diff
@@ -25,4 +25,10 @@ title: Foreman discovery documentation
 			<p id='manual'>4.0</p>
 		</a>
 	</div>
+  <div class='col-md-4 center'>
+		<a href="plugins/foreman_discovery/4.1/index.html" class="btn-doc btn">
+			<i class="fa fa-newspaper-o"></i>
+			<p id='manual'>4.1</p>
```

Comment:
Nitpick, this page is the opposite order to the Salt page, which goes latest -> oldest.